src/index.html

```diff
@@ -1,14 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+  </head>
+  <body>
+    <header class="header">
+      <a class="icon-ref" href="">
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+          class="icon"
+        >
+      </a>
+
+      <nav class="menu">
+        <a href="#apple" class="link is-active">APPLE</a>
+        <a href="#samsung" class="link ">SAMSUNG</a>
+        <a href="#smarts-and-cells" class="link ">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+        <a href=""
+          class="link "
+          data-qa="hover"
+        >
+          НОУТБУКИ И КОМПЬЮТЕРЫ
+        </a>
+        <a href="#gadgets" class="link">ГАДЖЕТЫ</a>
```

Comment:
```suggestion
        <a href="#gadgets" class="link">гаджеты</a>
```
make text uppercased using css